### PR TITLE
Create Aside component

### PR DESCRIPTION
**Pull request by renaudtertrais**
image

### CODE REVIEW

File: src/alto-ui/Aside/Aside.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { bemClass } from '../helpers/bem';
+
+import CloseButton from '../CloseButton';
+
+import './Aside.scss';
+
+function Aside({ className, onClose, show, title, children, a11yCloseLabel }) {
+  if (!show) return null;
+  return (
+    <aside className={bemClass('Aside', {}, className)}>
```
Comment (NicolasMartos) [inline src/alto-ui/Aside/Aside.js]: classnames ?

File: src/alto-ui/Aside/Aside.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      </div>
+      <div className="Aside__content">{children}</div>
+    </aside>
+  );
+}
+
+Aside.displayName = 'Aside';
+
+Aside.defaultProps = {
+  show: true,
```
Comment (NicolasMartos) [inline src/alto-ui/Aside/Aside.js]: false by default no?

File: src/alto-ui/Aside/Aside.js (hunk 1 above)
Comment (renaudtertrais) [reply src/alto-ui/Aside/Aside.js]: yes, I usually use `bemClass()` so it's ready to be used. As you prefer.

File: src/alto-ui/Aside/Aside.js (hunk 2 above)
Comment (renaudtertrais) [reply src/alto-ui/Aside/Aside.js]: nope: as it is optionnal (default props, not isRequired) then you can omit it but you want to display the panel anyway. Why ? perhaps you are doing the visibility yourself:

```jsx
{displayPanel && <Aside title="my panel">hello</Aside>}
```

**Pull request merged**